sdk/schemaregistry/schema-registry-avro/samples/javascript/sample.env

```diff
@@ -0,0 +1,10 @@
+SCHEMA_REGISTRY_ENDPOINT=<Endpoint URL>
+
```

Comment:
should we have a commented out entry for SCHEMA_REGISTRY_GROUP since that's used in the sample?